src/pages/Lending/hooks/useLendingCloseQuery.ts

```diff
@@ -82,6 +83,16 @@ const selectLendingCloseQueryData = memoize(
     const quoteInboundAddress = quote.inbound_address
     const quoteMemo = quote.memo
     const quoteExpiry = quote.expiry
+    const quoteOutboundDelayMs = bnOrZero(quote.outbound_delay_seconds).times(1000).toNumber()
+    const quoteInboundConfirmationMs = bnOrZero(quote.inbound_confirmation_seconds)
+      .times(1000)
+      .toNumber()
+    // Sane number for total time, in case the outbound is 0 seconds and the inbound is also fast,
```

Comment:
just curious if we have examples of txs where these values are 0?

Reply:
also would it be worth just using 0 or not showing the status bar if we can't estimate time to complete instead of an arbitrary 2 mins?

Reply:
Yeah, RUNE typically has 0 outbound delay, and depending on the state of the network, there may be 0 delay as well on other pools (saw it with all pools earlier today).
Note, that's for the outbound only and there will be smolish inbound_confirmation_seconds, but when borrowing from e.g ETH, those would only be something like 10s-20s. Making the total being 0 +20 = 20.

The reason why 2mn was chosen is because it's the interval time for polling with outbound checks enforced - it effectively means that by the time we show the Tx as complete, it *is* effectively complete. Just found it to be heuristics working perfectly well, vs. e.g 1mn which has high risks of missing confirmation on the subsequent poll following complete state (and often does).

I'd be inclined to leave it for consistency, and just because as a user, it's saner than only seeing a spinner!